src/hyperlight_host/tests/common/mod.rs

```diff
@@ -34,9 +35,12 @@ pub fn new_uninit() -> Result<UninitializedSandbox> {
 
 /// Use this instead of the `new_uninit` if you want your test to only run with the rust guest, not the c guest
 pub fn new_uninit_rust() -> Result<UninitializedSandbox> {
+    let mut cfg = SandboxConfiguration::default();
+    cfg.set_input_data_size(112 * 500 + 7); // 1 MiB
```

Comment:
Why do we need this set input data size?

Reply:
IIRC, this is a left over from debugging the issue that led me to introduce the `clear_io_buffers`.
Basically, if the guest function call ends abnormally, the io buffer doesn't get completely flushed.
The `interrupt_same_thread` test repeatedly calls a guest function that gets killed from the host, and eventually filled up the io buffer.
I'll revert this change.